bin/fixmystreet.com/one-off-status-update

```diff
@@ -140,7 +143,15 @@ if ($northants) {
         my $text = $report->{Response_to_Customer};
         # do not add a second response to customer if it's already there.
         if ($text) {
-            my $c = $p->comments->search({text => $text});
+            # sometimes there are extra spaces on the end in the database
+            my $search = { text => { ilike => "$text%" } };
+            # there's quite a few occurrences of the text being corrected that match this
+            if ( $text =~ /.+thank you[.\s]*$/i ) {
```

Comment:
No need for .+ at the start here. Though, why have the if at all, it won't remove anything if it's not present.